NuGet/EdFi.Security.DataAccess/53/EdFi.SecurityCompatiblity53.DataAccess/Contexts/PostgresSecurityContext.cs

```diff
@@ -3,58 +3,65 @@
 
-using System.Data.Entity;
-using System.Data.Entity.Core.Metadata.Edm;
-using System.Data.Entity.Infrastructure;
-using System.Data.Entity.ModelConfiguration.Conventions;
-using EdFi.Common;
+using System.Linq;
 using EdFi.Common.Utils.Extensions;
+using EdFi.SecurityCompatiblity53.DataAccess.Models;
+using Microsoft.EntityFrameworkCore;
 
 namespace EdFi.SecurityCompatiblity53.DataAccess.Contexts
 {
     public class PostgresSecurityContext : SecurityContext
     {
-        public PostgresSecurityContext(string connectionString)
-            : base(connectionString) { }
+        public PostgresSecurityContext(DbContextOptions options)
+            : base(options) { }
 
-        protected override void OnModelCreating(DbModelBuilder modelBuilder)
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
         {
             base.OnModelCreating(modelBuilder);
 
-            modelBuilder.Conventions.Add<ForeignKeyLowerCaseNamingConvention>();
-            modelBuilder.Conventions.Add<TableLowerCaseNamingConvention>();
+            modelBuilder.Model.GetEntityTypes().ForEach(entityType =>
+                entityType.SetSchema("dbo"));
 
-            modelBuilder.Properties().Configure(c => c.HasColumnName(c.ClrPropertyInfo.Name.ToLowerInvariant()));
-        }
+            //Fixes mapping error when using EFC
+            modelBuilder.Model.GetEntityTypes().Single(e => e.ClrType.Name == nameof(ClaimSet))
+                .GetProperty("ApplicationId")
+                .SetColumnName("application_applicationid");
 
-        private class TableLowerCaseNamingConvention : IStoreModelConvention<EntitySet>
-        {
-            public void Apply(EntitySet entitySet, DbModel model)
-            {
-                Preconditions.ThrowIfNull(entitySet, nameof(entitySet));
-                Preconditions.ThrowIfNull(model, nameof(model));
+            modelBuilder.Model.GetEntityTypes().Single(e => e.ClrType.Name == nameof(ClaimSetResourceClaim))
+                .GetProperty("ResourceClaimId")
+                .SetColumnName("resourceclaim_resourceclaimid");
 
-                entitySet.Table = entitySet.Table.ToLowerInvariant();
-            }
-        }
+            modelBuilder.Model.GetEntityTypes().Single(e => e.ClrType.Name == nameof(ClaimSetResourceClaim))
+                .GetProperty("ActionId")
+                .SetColumnName("action_actionid");
 
-        private class ForeignKeyLowerCaseNamingConvention : IStoreModelConvention<AssociationType>
-        {
-            public void Apply(AssociationType association, DbModel model)
-            {
-                Preconditions.ThrowIfNull(association, nameof(association));
-                Preconditions.ThrowIfNull(model, nameof(model));
+            modelBuilder.Model.GetEntityTypes().Single(e => e.ClrType.Name == nameof(ClaimSetResourceClaim))
+                .GetProperty("ClaimSetId")
+                .SetColumnName("claimset_claimsetid");
```

Comment:
NET8.0 migration makes this type of changes necessary to map this type of fields.